engine/src/main/java/org/terasology/rendering/dag/nodes/BackdropNode.java

```diff
@@ -87,7 +90,7 @@ public void process() {
      */
     // It's unclear why these buffers need to be cleared while all the others don't...
     private void initialClearing() {
-        sceneReflectiveRefractive = frameBuffersManager.getFBO("sceneReflectiveRefractive");
+        sceneReflectiveRefractive = dynamicFBM.getFBO(REFRACTIVE_REFLECTIVE_URN);
```

Comment:
Similarly to comment above, the functionality for this to NOT happen within process() should now be available.